Services/Interfaces/web_interface/static/js/components/config_tentacle.js

```diff
@@ -61,12 +61,26 @@ function handle_tentacle_config_update_error_callback(updated_data, update_url,
     create_alert("error", "Error when updating config", msg.responseText);
 }
 
+function check_config(){
+    var errors = configEditor.validate();
+
+    if(errors.length)
+        return false
+    else
+    const errors = configEditor.validate();
+    return !errors.length;
```

Comment:
I'm not sure I understand why calling twice `configEditor.validate()` ?
To me this 2 lines should be enough to validate the form and return a boolean corresponding to whether there are errors or not (basically the whole `check_config` function):
```python
const errors = configEditor.validate();    
return !errors.length;
```

Reply:
Sorry, I just applied the code review without looking